libs/sdk-ui-charts/src/interfaces/chartConfig.ts

```diff
@@ -359,3 +364,15 @@ export interface IAxisConfig {
      */
     name?: IAxisNameConfig;
 }
+
+/**
+ * Tooltip customization
+ *
+ * @public
+ */
+export interface ITooltipConfig {
+    /**
+     * Indicates if tooltip is shown.
```

Comment:
Please document the default value of this. You can use `@defaultValue` tag for it.